<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1319</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Quasar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupConectar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>231</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Status da conexão</string>
    </property>
    <widget class="QPushButton" name="buttonConectar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Conectar</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Desconectado</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupConf">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>401</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Configuração</string>
    </property>
    <widget class="QGroupBox" name="groupSinal">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>221</width>
       <height>211</height>
      </rect>
     </property>
     <property name="title">
      <string>Sinal</string>
     </property>
     <widget class="QDoubleSpinBox" name="dSpinAmp">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>80</y>
        <width>68</width>
        <height>23</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="labelAmp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Amplitude</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelOffSet">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Off-set</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dSpinOffSet">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>110</y>
        <width>68</width>
        <height>23</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="labelPeriodo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Período</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dSpinPeriodo">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>68</width>
        <height>23</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QComboBox" name="comboTipoSinal">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="labelTipoSinal">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>59</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Tipo:</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="dSpinAux">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>170</y>
        <width>68</width>
        <height>23</height>
       </rect>
      </property>
      <property name="singleStep">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="labelAux">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="buttonAtualizar">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>210</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Atualizar</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupControl">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>151</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Controle</string>
     </property>
     <widget class="QRadioButton" name="radioAberta">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>102</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Malha aberta</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioFechada">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Malha fechada</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="graficoEscrita" native="true">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>891</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_tensaoEscrita">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_tensaoCalculada">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QCustomPlot" name="graficoLeitura" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>1301</width>
      <height>271</height>
     </rect>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lb_sinalLido">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lb_erro">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1319</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfigura_es">
    <property name="title">
     <string>Configurações</string>
    </property>
    <addaction name="actionCanal"/>
   </widget>
   <addaction name="menuConfigura_es"/>
  </widget>
  <action name="actionCanal">
   <property name="text">
    <string>Canal</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
